tests/acceptance/features/webUILogin/adminBlocksUser.feature

```diff
@@ -14,3 +15,14 @@ Feature: view profile
     Then the user should be redirected to the login error page
     When the user exits the login error page
     Then the user should be redirected to the login page
+
+  @oauthLogin @notToImplementOnOCIS
```

Comment:
why shouldn't that be implemented on OCIS?

Reply:
ocis doesn't support oauth2 login, it comes with openid by default